source/CardGames.py

```diff
@@ -346,7 +346,10 @@ def display_board_general(self):
               card = stack[0]
             else:
               card = stack
-            image = card.image[idx] if card.revealed else card.cardBack[idx]
+            if player == 'fugitive' or card.revealed:
```

Comment:
Looks great here! I'm assuming that when we call the function, we need to explicitly set the player parameter as fugitive?

Reply:
Thanks! Yes, if its the fugitive's turn then it has to be passed exactly as 'fugitive'